.github/workflows/win-cuda-x64-build.yml

```diff
@@ -81,7 +81,10 @@ jobs:
     - name: Install the Python Wheel and Test Dependencies
       run: |
         python -m pip install -r test\python\requirements.txt
-        python -m pip install -r test\python\requirements-cuda.txt
+        python -m pip install -r test\python\requirements-torch-cuda.txt
+        python -m pip install -r test\python\requirements-ort-cuda.txt
+        # python -m pip install -f https://download.pytorch.org/whl/torch_stable.html torch==2.2.1+cu121
+        # python -m pip install --pre -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ ort-nightly-gpu==1.20.0.dev20241007001
```

Comment:
You may consider upgrading the nightly package to a newer version.  Recently we changed the package name, also the feed URL.

Reply:
@snnn can you share where can I find the new feed URLs?

Reply:
CUDA 12 packages are still under the same URL. But package name is changed to onnxruntime. 
See https://github.com/microsoft/onnxruntime/pull/22529

Reply:
Thanks for sharing this, I believe we can update the version in a separate PR to avoid any another issues, will share the details with the team.

Reply:
You need to. 

![image](https://github.com/user-attachments/assets/c2fe3771-eeb3-4a04-a501-ad0660ff57be)

Reply:
Here you installed ort-nightly first, then when you install onnxruntime_genai, then genai package says it depends on onnxruntime instead of ort-nightly. But you didn't install onnxruntime. You only have ort-nightly.